### PR TITLE
Fix: Throw error when undefined viewing key added (#23)

### PR DESCRIPTION
**Pull request by David-Sod**
Heres a simple fix for undefined and empty VKs. Now it should throw and error with the following message "Empty or undefined key cannot be added"

### CODE REVIEW

File: src/viewing-keys.ts [hunk 1]
```diff
@@ -41,6 +41,7 @@ export class ViewingKeyManager {
       account = this.addAccount();
     }
     if (!account) throw new Error('No account available');
+    if (!key) throw new Error('Empty or undefined key cannot be added');
```
Comment (luis-stakeordie) [inline src/viewing-keys.ts:44]: Should be this the first thing that you validate in this function?

**Review by luis-stakeordie — CHANGES_REQUESTED** [review]
@David-Sod check out this comments, fix them should be straightforward.

File: src/viewing-keys.ts (hunk 1 above)
Comment (David-Sod) [reply src/viewing-keys.ts]: My bad, moving this validation as firts line of the function

Comment (luis-stakeordie) [reply src/viewing-keys.ts]: Thanks

**Review by luis-stakeordie — APPROVED** [review]
Yep, there it is.

**Pull request merged**